src/main/java/org/jenkinsci/main/modules/sshd/SSHD.java

```diff
@@ -83,18 +94,40 @@ public void run() {
         }
     }
 
+    /**
+     * Provides a list of Cipher factories, which can be activated on the instance.
+     * Cyphers will be considered as activated if they are defined in {@link #ENABLED_CIPHERS} and supported in the current JVM.
+     * @return List of factories
+     */
+    @Nonnull
+    /*package*/ static List<NamedFactory<Cipher>> getActivatedCiphers() {
+        final List<NamedFactory<Cipher>> activatedCiphers = new ArrayList<>(ENABLED_CIPHERS.size());
+        for (NamedFactory<Cipher> cipher : ENABLED_CIPHERS) {
+            if (cipher instanceof BuiltinCiphers) {
+                final BuiltinCiphers c = (BuiltinCiphers)cipher;
+                if (c.isSupported()) {
+                    activatedCiphers.add(cipher);
+                } else {
+                    LOGGER.log(Level.WARNING, "Discovered unsupported built-in Cipher: {0}. It won't be enabled", c);
+                }
+            } else {
+                // We cannot determine if the cipher is supported, but the default configuration lists only Built-in ciphers.
+                // So somebody explicitly added it on his own risk.
```

Comment:
It should happen once during the SSHD startup, so I do not think it is a big deal. But the problem is that these errors will appear in Stock Java builds, hence downgrading to FINE is probably a good approach